dallinger/frontend/templates/dashboard_cli.html

```diff
@@ -0,0 +1,48 @@
+{% extends "base/dashboard.html" %}
+
+{% block body %}
+<h1>Experiment lifecycle dashboard</h1>
+
+    <h2>Shut down experiment</h2>
+    
+    <pre>dallinger destroy --app {{ id }}</pre><button class="copy-button">Copy</button>
+
+    <h2>Export data</h2>
+    <pre>dallinger export --app {{ id }}</pre><button class="copy-button">Copy</button>
+
+    
+    <script>
+        function copy_text (source) {
+            var copied = false;
+            const textarea = document.createElement('textarea');
+            textarea.value = source;
+            textarea.style.position = 'absolute';
+            textarea.style.left = '-1000000px';
+            document.body.appendChild(textarea);
+            textarea.select()
+            
+            try {
+                var successful = document.execCommand('copy');
```

Comment:
Should we pull in a [library](https://clipboardjs.com/) for this since browser support is spotty? My impression is that this won't work in Safari, MSIE, and all but the most recent versions of MS Edge.

Reply:
I tried using the `navigator.clipboard` api, and it doesn't pass the window.isSecureContext test even on Heroku, so I'm now open to clipboard.js.

Reply:
Just tried clipboard.js and it was quite nice.